tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -630,6 +630,8 @@ def _screenshot(self, protocol, url, timeout):
         self.protocol.base.execute_script(self.wait_script, True)
 
         screenshot = self.protocol.webdriver.screenshot()
+        if screenshot is None:
+            raise error.UnknownErrorException()
```

Comment:
It seems odd to return a webdriver error here, given that there apparently wasn't a protocol-level error, but we somehow got a `None` result (maybe the browser crashed?). Can we at least just raise a normal Python exception like `ValueError`?

Reply:
updated, thanks!